### DISCUSSION TITLE
#65 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0x-stan** [tog-ETH-er/togETHer#50] [General]
Agenda:

Community Updates
Fireside Chat
会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/uvj-fdcj-wvj

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

3个PR #966 ～ #968

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会
 
- 《Curve Stablecoin 白皮书数学推导》by 0xmc
  - Youtube: https://www.youtube.com/watch?v=Od-UYMPxsKg
  - Bilibili: https://www.bilibili.com/video/BV1Hu411Y7xW/

- 《Morpho 项目解读》by yyy
  - Youtube: https://youtu.be/bRJKlk6M4QE
  - Bilibili: https://www.bilibili.com/video/BV1po4y1E7xr

**Comment by luckyyang** [comment]
Rebase Community

- web3 极客日报: 最近 14 期 http://daily.rebase.network/
- 招聘: 1 个团队发布招聘 https://github.com/rebase-network/who-is-hiring/issues
- web3极客公开课（web3geekforge ）1 期 分享 + 1 期 twitter space: 
  - https://space.bilibili.com/382886213/video
  - https://twitter.com/i/spaces/1BdxYyQgRNMxX
- 2 个活动 https://github.com/rebase-network/event-pool/issues

**Comment by Riely007** [comment]
**以太坊翻译小组**

- 日常 review 进行中

**Comment by bambalados** [comment]
**Crypto Tech Night**

-

**Comment by Stamp9** [comment]
### 原语里弄

研究进行中。
